source/includes/_tipser-elements-analytics.md

```diff
@@ -1,189 +1,308 @@
----------        | -----------    | -----------               
-`E-commerce`     | `checkout`     | Yes (with one or many products attached)
-
-
-## Purchase
-
-This event is emitted when a product (or several products) is purchased.
-
-[![](purchase_event.png)](/images/purchase_event.png)
-
-_In the screenshot above the user has finalized the payment and the order confirmation dialog has been displayed. "Purchase" event has been emitted._
-
-### GA event format ###
-
-Category         | Action         | Standard Enhanced Ecommerce event?        
----------        | -----------    | -----------               
-`E-commerce`     | `checkout`     | Yes (with one or many products attached)
+# Tipser Elements Analytics
+
+Tipser has implemented an event for the Tipser Elements Analytics that users of the script can listen to in order to gather analytics about Tipser interactions and feed them into their own analytics engines.
+
+## General overview of the Tipser Elements Analytics collector
+By listening to the event `tw-track` certain Tipser events can be gathered and pushed into an external analytics engine.
+
+The event contains the object detail that contains the following items:
+
+`detail` object
+
+```yaml
+{
+  action: string, # what action is taken on the target, mandatory
+  target: string, # where this action is taken, mandatory
+  description: string, # describing what is being logged
+  object: # details about the tracked object, e.g. collection ID, product ID, etc.
+  [    
+    {  
+      # some properties here               
+    },     
+    ..., # this array can contain more than 1 object
+  ]
+}
+```
+
+
+Example of an event listener:
+
+```javascript
+document.addEventListener('tw-track', function (e) {
+  var detial = e.detail; 
+  console.log('tw-track');
+  console.log('tw-track action',detial.action);
+  console.log('tw-track description',detial.description);
+  console.log('tw-track target',detial.target);
+  console.log('tw-track object',detial.object);
+});
+```
+
+## List of supported events
+
+### View collection
+When a collection is in a viewport.
+
+`detail` object structure
+
+ ```yaml
+{
+  description: ‘Viewport scrolled’,
+  action: 'View',
+  target: 'List',
+  object:  {
+    id: string,
+    ownerUserId: string,
+    created: string, #e.g. DateTime ISO 2019-06-11T08:40:29.377Z
+    modified: string, #e.g. DateTime ISO 2019-06-11T08:40:29.377Z
+    postComment: null | ?
+    product: Product     
+  }[]
+}
+```  
+_Quick links to object structures: [Product](#product-structure)_ 
+
+### Click product in collection
+When a product in a collection is clicked.
+
+`detail` object structure
+
+```yaml
+{
+  description: 'Product tile clicked',
+  action: 'Click',
+  target: 'Product tile',
+  object: Product[]
+}
+```
+_Quick links to object structures: [Product](#product-structure)_
+
+### View product
+When a product in a collection is viewed.
```

Comment:
It seems to me that the description below refers to a different event ("product details viewed" instead of "product viewed"). Could you please double check it?

Reply:
Yes, thanks!